packages/suite/src/hooks/wallet/useCoinmarketBuyOffers.ts

```diff
@@ -43,17 +43,18 @@ export const useOffers = (props: Props) => {
         setIsFromRedirect,
         openCoinmarketBuyConfirmModal,
         addNotification,
+        saveTransactionDetailId,
     } = useActions({
         saveTrade: coinmarketBuyActions.saveTrade,
         setIsFromRedirect: coinmarketBuyActions.setIsFromRedirect,
         openCoinmarketBuyConfirmModal: coinmarketBuyActions.openCoinmarketBuyConfirmModal,
         addNotification: notificationActions.addToast,
+        saveTransactionDetailId: coinmarketBuyActions.saveTransactionDetailId,
     });
 
-    const invityAPIUrl = useSelector<
-        AppState,
-        AppState['suite']['settings']['debug']['invityAPIUrl']
-    >(state => state.suite.settings.debug.invityAPIUrl);
+    const invityAPIUrl = useSelector<AppState['suite']['settings']['debug']['invityAPIUrl']>(
```

Comment:
you don need to use type anymore, it should be typed already:
`const invityAPIUrl = useSelector(state => state.suite.settings.debug.invityAPIUrl);`